src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -187,6 +187,17 @@
                                                     <div class="tab">
                                                         <h3 class="tab-title">Attachment #{{ loop.index }}</h3>
                                                         <div class="tab-content">
+                                                            {% set path = collector.path(attachment) %}
+                                                            {% if path is not null %}
+                                                                {% set filename = collector.filename(attachment) %}
+
+                                                                <p><a href="{{ path('_profiler_mailer_attachment', {
+                                                                    path: path,
+                                                                    filename: filename
+                                                                }) }}" target="_blank">
+                                                                    {{ filename }}
+                                                                    </a></p>
+                                                            {% endif %}
```

Comment:
Assuming the embed works and I'm not missing something, I think we should add `DataPart::getFilename()/getContentType()` in 6.1. Then, all that would be required is something like this:

```twig
{% set filename = attachment.filename|default %} {# symfony/mime < 6.1 would resolve to null #}

{% if filename %}
<p>
    <a href="data:{{ attachment.contentType }};base64,{{ collector.base64Encode(attachment.body) }}" download="{{ filename }}">{{ filename }}</a>
</p>
{% endif %}
```